ci-bench-runner/src/event_queue.rs

```diff
@@ -75,7 +75,7 @@ impl EventQueue {
                     octocrab.clone(),
                 );
 
-                info!("background task started for event queue");
+                info!("background task started for event queue job {active_job_id:?}");
```

Comment:
I'd expect the active job's id here won't provide meaningful information (it gets reset on every loop iteration and will always be `None` at the beginning of the loop). If the goal is to log the job id when a job starts we should do it in `process_queued_events_in_background`. Is that what you meant?

Reply:
> If the goal is to log the job id when a job starts we should do it in process_queued_events_in_background. Is that what you meant?

That sounds more useful :-) I think I wasn't thinking about it very deeply. I'll put up a fix.